### PR TITLE
fix: Add explicit imports to prevent bundling issues

### PR DESCRIPTION
**Pull request by guibulator**
Added explicit imports to fix issues when consuming this module as external dependency.

### CODE REVIEW

**Comment by hmaack** [comment]
This fix works for me, the current main throws unfortunately missing function `createOpenFetch` exceptions because of this autoimport. @enkot what do you think?

File: .nuxtrc [hunk 1]
```diff
@@ -1 +1,2 @@
 typescript.includeWorkspace=true
+imports.autoImport=false
```
Comment (IlyaSemenov) [inline .nuxtrc:2]: The project has `.editorconfig` which explicitly forces final new line:

https://github.com/enkot/nuxt-open-fetch/blob/55a619189adde50a418c64d1d1da1bd93c0bdc9c/.editorconfig#L9

Please lint your code accordingly.

Comment (guibulator) [reply .nuxtrc]: Done

**Comment by enkot** [comment]
Thank you for the PR!

**Pull request merged**